app/src/main/AndroidManifest.xml

```diff
@@ -53,6 +53,16 @@
 
                 <data android:scheme="wc" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
```

Comment:
You can just put this `<data>` clause inside the existing `intent-filter` above.